src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -3525,6 +3528,18 @@ compiler.err.module.not.found=\
 compiler.warn.module.not.found=\
     module not found: {0}
 
+# 0: name
+compiler.err.import.module.not.found=\
+    imported module not found: {0}
+
+# 0: symbol
+compiler.err.import.module.does.not.read.unnamed=\
+    unnamed module does not read: {0}
+
+# 0: symbol, 1: symbol
+compiler.err.import.module.does.not.read=\
+    {1} module does not read: {0}
```

Comment:
shouldn't it be `module {1} does not read: {0}` ? Also, maybe worth reordering the params?

Reply:
Thanks! Fixed:
https://github.com/openjdk/jdk/pull/18614/commits/846b038ed8145ede9c419daddedb794a429dafac